### PR TITLE
:wrench: Disable HTTP request logs when running tests

### PR DESCRIPTION
**Pull request by DronePT**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "nodemon src/index.js",
     "build": "sucrase ./src -d ./dist --transforms imports",
-    "test": "mocha --require sucrase/register ./test/**/*.js"
+    "test": "NODE_ENV=test mocha --require sucrase/register ./test/**/*.js"
```
Comment (masterl) [inline package.json:9]: Since we are using `dotenv` this value should be set on `.env`

**Review by masterl — CHANGES_REQUESTED** [review]
Environment variables should be set on `.env`
check `example.env`

File: package.json (hunk 1 above)
Comment (DronePT) [reply package.json:9]: I've removed it from package.json and I did write how to run the tests on README file.

Comment (joviane) [reply package.json:9]: The common convention is to set NODE_ENV in your build script and not in .env file so @DronePT was right to set in `package.json`. If you set in ,env you'll have to remeber to alter this variable every time that you wants to change the environment and possible having some side effects.

Comment (joviane) [reply package.json:9]: Could you please go back to your original version @DronePT?

Comment (DronePT) [reply package.json:9]: @joviane setting NODE_ENV on test script is what I'm used to do also. I've reverted the branch to the last commit. 

Thank you

**Review by joviane — APPROVED** [review]
:wrench: done

**Pull request merged**